frontend/now.json

```diff
@@ -0,0 +1,7 @@
+{
+    "type": "npm",
```

Comment:
is there a yarn? because npm wouldn't look at yarn.lock

Reply:
Now supports 3 types of deployments: static, npm, and docker. npm vs yarn is just a package manager difference. This does mean though that internally it calls "npm install" and generates a "package-lock.json" file. The other solution will be making it a docker deployment. I can look into doing this, and possibly make another PR.